tests/FacebookClientTest.php

```diff
@@ -70,23 +70,11 @@ public function testACustomHttpClientCanBeInjected()
     {
         $handler = new MyFooClientHandler();
         $client = new FacebookClient($handler);
-        $httpHandler = $client->getHttpClientHandler();
+        $httpHandler = $client->getHttpClient();
 
         $this->assertInstanceOf('Facebook\Tests\Fixtures\MyFooClientHandler', $httpHandler);
     }
 
-    public function testTheHttpClientWillFallbackToDefault()
```

Comment:
Please keep a test that checks that without constructor parameters, we get an instance of `Http\Client\HttpClient`